grafana/overlays/nerc-ocp-infra/grafanadatasources/observability-metrics.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDataSource
+metadata:
+  name: observability-metrics
+  namespace: grafana
+spec:
+  name: observability-metrics
+  datasources:
+    - name: observability-metrics
+      access: proxy
+      editable: false
+      isDefault: true
+      jsonData:
+        httpHeaderName1: Authorization
+        timeInterval: 5s
+        tlsAuthWithCACert: true
+      secureJsonData:
+        httpHeaderValue1: "Bearer ${token}"
+        tlsCACert: "${service-ca.crt}"
+      type: prometheus
+      url: 'https://thanos-querier.openshift-monitoring.svc.cluster.local:9091/'
```

Comment:
@harshil-codes pointed out that we should change the url here to be the correct observability thanos querier `http://observability-thanos-query.open-cluster-management-observability.svc.cluster.local:9090`

Reply:
@schwesig @larsks  I just added a commit to change this URL while making another change to add a ClusterRoleBinding for the grafana-servicaccount to read the metrics.